### ISSUE TITLE
Any plans on supporting .yaml or .json format for specifying the command line args.

### ISSUE THREAD

**Issue by srsaggam** [dotnet/command-line-api#454]
It seems right now only .rsp is the alternative format to specify the command line args. Since most of the new projects follow the .json or .yaml formats. Is there a good story around enabling these kind of formats?

**Comment by srsaggam** [comment]
@jonsequitur 
cc: @CESARDELATORRE @vinodshanbhag  @rjoshi

**Comment by CESARDELATORRE** [comment]
Agree. Here's one example of a CLI in the market using a .YAML file as the main argument:

https://eng.uber.com/introducing-ludwig/

https://uber.github.io/ludwig/examples/

**Comment by srsaggam** [comment]
cc: @justinormont
